plugins/console-logger/ios/Sources/PrintLoggerPlugin.swift

```diff
@@ -35,10 +35,14 @@ public class PrintLoggerPlugin: NativePlugin {
     public func apply<P>(player: P) where P: HeadlessPlayer {
         player.logger.logLevel = logLevel
         player.logger.hooks.prefixMessage.tap(name: pluginName) { .bail("[Player] [\($0.description)]: ") }
-        player.logger.hooks.trace.tap(name: pluginName, { print($0) })
-        player.logger.hooks.debug.tap(name: pluginName, { print($0) })
-        player.logger.hooks.info.tap(name: pluginName, { print($0) })
-        player.logger.hooks.warn.tap(name: pluginName, { print($0) })
-        player.logger.hooks.error.tap(name: pluginName, { print($0 ?? "", $1?.localizedDescription ?? "") })
+
+        let prefixedMessage = player.logger.hooks.prefixMessage.call(logLevel) ?? ""
```

Comment:
i think we can remove the `prefixMessage` hook from `TapableLogger`, and just have that be opinionated behavior of `PrintLoggerPlugin`